test/mocks/transformedAnalysisData.js

```diff
@@ -46,6 +47,7 @@ const transformedAnalysisData = {
       pmFileName: '',
       analysisFileName: 'analysis-SF1624HT2008_1.pdf',
       roundIdList: '1',
```

Comment:
`ladokRoundIds ` and `roundIdList` has to be removed everywhere, in order to be sure tests work

Reply:
ladokRoundIds and roundIdList was removed from everywhere